templates/layouts/base.tera

```diff
@@ -14,6 +14,7 @@
     <meta name="description" content="{{ description }}" />
     <meta name="keywords" content="Isabel Roses, blog, go, rust, personal, tech, programming, web development, software engineering" />
     <link rel="canonical" href="https://isabelrose.com">
+    <link rel=expect href="#below-the-fold" blocking=render>
```

Comment:
This one ensures that view transitions do not start to early. I the fragment identifier is not defined, the page loads completely before the view transition starts. For a long page, you can define this like `<a name="below-the-fold"/>` somewhere below the fold.